waddrmgr/scoped_manager.go

```diff
@@ -1927,9 +1927,6 @@ func (s *ScopedKeyManager) ImportPrivateKey(ns walletdb.ReadWriteBucket,
 		return nil, managerError(ErrWrongNet, str, nil)
 	}
 
-	s.mtx.Lock()
```

Comment:
Are you sure we can remove this lock?

Reply:
I removed this change, so resolving the conversation